.github/workflows/publish.yml

```diff
@@ -79,14 +85,10 @@ jobs:
           echo "SDKROOT=$(xcrun -sdk macosx$(sw_vers -productVersion) --show-sdk-path)" >> $GITHUB_ENV
           echo "MACOSX_DEPLOYMENT_TARGET=$(xcrun -sdk macosx$(sw_vers -productVersion) --show-sdk-platform-version)" >> $GITHUB_ENV
 
-      - uses: actions/cache@v3
+      - uses: actions/cache/restore@v3
+        id: restore-cache
```

Comment:
Thoughts on naming these just "cache"? Would make for cleaner use of the outputs